packages/cli-v3/src/entryPoints/deploy-index-worker.ts

```diff
@@ -115,6 +115,20 @@ if (typeof config.machine === "string") {
   });
 }
 
+// If the config has a maxDuration, we need to apply it to all tasks that don't have a maxDuration
```

Comment:
What if you wanted to have a default maxDuration but then have a specific task not have a maxDuration? It would be nice if you could pass `null` to say you don't want a limit. Or maybe some const we provide, like `MaxDuration.None` (that is just Math.Infinity).